packages/koa-metrics/src/timing.ts

```diff
@@ -0,0 +1,29 @@
+import {StatsD} from 'hot-shots';
+import {Context} from 'koa';
+
+export interface Timer {
+  close(responseClient?: StatsD): void;
```

Comment:
I think close is a bit weird of a name. If I was going to time something on a stopwatch or my phone I would probably say `end` or `stop` instead of `close` to describe finishing timing.